packages/rollup-plugin-index-html/rollup-plugin-index-html.js

```diff
@@ -73,6 +76,36 @@ module.exports = (pluginConfig = {}) => {
       return result.rollupOptions;
     },
 
+    resolveId(source, importer) {
+      const { rootDir } = localPluginConfig;
+      const relativeSource = source.replace(rootDir, '');
+
+      const basePath = importer ? importer.replace(rootDir, `${rootDir}::`) : `${rootDir}::`;
+      if (!importMapCache) {
+        // const mapString = fs.readFileSync(path.join(currentDir, 'import-map.json'), 'utf-8');
```

Comment:
@LarsDenBakker do you have an idea how to best get the import-map string from the index.html?

also we might need an option to remove the import-map from the generated index.html?